doc/Makefile.docfiles

```diff
@@ -0,0 +1,59 @@
+#**************************************************************************
+#*                                                                        *
+#*                                 OCaml                                  *
+#*                                                                        *
+#*             Florian Angeletti, projet Cambium, Inria Paris             *
+#*                                                                        *
+#*   Copyright 2020 Institut National de Recherche en Informatique et     *
+#*     en Automatique.                                                    *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+
+define capitalize
+$(shell echo $(1) | sed "s/\<./\U&/g")
```

Comment:
Does this magic sed stuff `sed "s/\<./\U&/g"` work on exotic systems with old Sed versions? (@Octachron and myself wondered if we should use a small OCaml script instead, but properly dealing with spaces-in-directories might be trickier.)

Reply:
> Does this magic sed stuff `sed "s/\<./\U&/g"` work on exotic systems with old Sed versions? 

The `\U` replacement (meaning "convert to uppercase", I guess) is no documented in POSIX sed, so the answer is no, it's not going to work everywhere.  And it's the first time I see this replacement!

Reply:
Here is a classic sed implementation that I've been using since the 90's:
```
sed -e 'h
s/\(.\).*/\1/
y/abcdefghijklmnopqrstuvwxyz/ABCDEFGHIJKLMNOPQRSTUVWXYZ/
G
s/\n.//'
```

Reply:
Yes, the good sed scripts are from the 90s :-)

Reply:
Here is a sed-free solution:
```
define capitalize
$(shell echo $(1) | cut -c1 | tr '[:lower:]' '[:upper:]')$(shell echo $(1) | cut -c2-)
endef
```

Reply:
Then let's go for the recommended solution and a bit of glue rather than a GNUism.